code/workspaces/web-app/src/components/common/ComponentPermissionWrapper.js

```diff
@@ -1,10 +1,11 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { SYSTEM_INSTANCE_ADMIN } from 'common/src/permissionTypes';
 
 class ComponentWrapper extends Component {
   hasPermission() {
     const { userPermissions, permission } = this.props;
-    return userPermissions.includes(permission);
+    return userPermissions.includes(permission) || userPermissions.includes(SYSTEM_INSTANCE_ADMIN);
```

Comment:
Did we ever decide if a system admin should be able to see everything? If they shouldn't then this might not be quite right?

Reply:
I think that's the plan, at least for the moment - @gary-lloyd-tessella ?

Reply:
I think for now I’m happy for an instance admin to see everything.